### PR TITLE
[strings] Update editor_edits_sent_message string

### PR DESCRIPTION
**Pull request by MontakOleg**
Update `editor_edits_sent_message` string to reflect real app behavior (updates will be sent when the app goes to the background), as suggested in comment https://github.com/organicmaps/organicmaps/pull/6986#discussion_r1437008252

### CODE REVIEW

**Review by biodranik — APPROVED** [review]
Thanks! Calling for reviews from @organicmaps/translations

File: data/strings/strings.txt
Comment (fitojb) [inline data/strings/strings.txt:16397]: Let's remove the es-MX variant and edit es as follows: "Su nota se enviará a OpenStreetMap".

File: data/strings/strings.txt
Comment (fitojb) [inline data/strings/strings.txt:16391]: "vostra" instead of "teva", pls

**Review by fitojb — CHANGES_REQUESTED** [review]

File: data/strings/strings.txt
Comment (vgrigoriu) [inline data/strings/strings.txt:16417]: Romanian translation sounds good, I'm just curious about the vocabulary (across all languages): is this a note, a comment, a message?

File: data/strings/strings.txt
Comment (MontakOleg) [reply data/strings/strings.txt:16391]: updated

File: data/strings/strings.txt
Comment (MontakOleg) [reply data/strings/strings.txt:16397]: updated

File: data/strings/strings.txt
Comment (biodranik) [reply data/strings/strings.txt:16417]: @vgrigoriu can you please check how is it called at osm.org? It's a note/comment to OSM editors what should be checked/fixed in the maps data.

Comment (vgrigoriu) [reply data/strings/strings.txt:16417]: Yep, it's "note" on osm.org.

**Review by jimcarst — APPROVED** [review]
ok for nl

File: data/strings/strings.txt
Comment (proxygeek) [reply data/strings/strings.txt:16417]: "Hi" is good

**Review by proxygeek — APPROVED** [review]

File: data/strings/strings.txt
Comment (matheusgomesms) [inline data/strings/strings.txt:16415]:
```suggestion
    pt = A sua nota será enviada ao OpenStreetMap
```

Comment (matheusgomesms) [reply data/strings/strings.txt:16415]: PT and PT-BR equal here

Comment (MontakOleg) [reply data/strings/strings.txt]: updated

File: iphone/Maps/LocalizedStrings/hu.lproj/Localizable.strings
Comment (MontakOleg) [reply iphone/Maps/LocalizedStrings/hu.lproj/Localizable.strings:814]: Hm, DeepL says that `Az Ön változtatási javaslata elküldésre került` means `Your change proposal has been sent`.

But we want to emphasize that change is not sent yet, it will be sent at some time in the future.

Comment (MontakOleg) [reply iphone/Maps/LocalizedStrings/hu.lproj/Localizable.strings:814]: @d4f5409d happy new year! Waiting for your response here.

Comment (d4f5409d) [reply iphone/Maps/LocalizedStrings/hu.lproj/Localizable.strings:814]: HNY! Don't rely on DeepL that much, it isn't 100% perfect ;)
I changed the translation accordingly to your review.

Comment (MontakOleg) [reply iphone/Maps/LocalizedStrings/hu.lproj/Localizable.strings]: updated ✔️

**Comment by MontakOleg** [comment]
@biodranik hi! Are we waiting for more reviews or can proceed with merging this?

**Comment by Jean-BaptisteC** [comment]
@biodranik PTAL

**Comment by Jean-BaptisteC** [comment]
@MontakOleg can you rebase your branch?

**Comment by MontakOleg** [comment]
@Jean-BaptisteC yep, rebased.

**Comment by Jean-BaptisteC** [comment]
@biodranik ready to merge

**Comment by biodranik** [comment]
@MontakOleg can you please squash/fixup two last commits before the merge?

**Comment by MontakOleg** [comment]
@biodranik Sure! Done.

**Pull request merged**
